src/openseadragon.js

```diff
@@ -187,6 +187,9 @@
   * @property {Number} [opacity=1]
   *     Default opacity of the tiled images (1=opaque, 0=transparent)
   *
+  * @property {Boolean} [preload=false]
+  *     If true, tiles in image load without drawing. Defaults to false, so tiles are loaded and drawn.
```

Comment:
I'm confused. Yes- `preload: true` will make the tiles not draw. That's the goal- if `preload==true`, the tiles won't draw until someone switches `preload==false`...
This was my intention:
|                | `preload == true` | `preload == false` |
|----------------|-------------------|--------------------|
| `opacity == 0` | Do Nothing        | Do Nothing         |
| `opacity > 0`  | Load only         | Load & Draw        |

I think this is what you have in mind:
|                | `preload == true` | `preload == false` |
|----------------|-------------------|--------------------|
| `opacity == 0` | Load only         | Do Nothing         |
| `opacity > 0`  | Load only         | Load & Draw        |